gdmix-workflow/images/gdmix/Dockerfile

```diff
@@ -5,11 +5,10 @@ USER root
 # Set default shell to /bin/bash
 SHELL ["/bin/bash", "-cu"]
 
-ENV GDMIX_VERSION=0.3.0
+ENV GDMIX_WORKFLOW_VERSION=0.4.0
```

Comment:
Please don't change the Dockerfile for now, this will be done after publishing:
-  a new wheel version of GDMix trainer and workflow
-  a new jar version of GDMix data

Reply:
Yeah, let's bump the version later.
For the artifacts: Trainer related wheel and jars stay the same due to the decoupling. Please see the other comment.

Reply:
It looks like that the dockerfile explicitly references the trainer version. I'll either revert or think of something else.

Reply:
reverted.